src/getNonce.ts

```diff
@@ -0,0 +1,8 @@
+export function getNonce(): string {
```

Comment:
Can we use the crypto library for generating this?
Could just use `crypto.randomUUID()`:
https://nodejs.org/docs/latest-v16.x/api/crypto.html#cryptorandomuuidoptions